js/config.js

```diff
@@ -119,12 +119,27 @@ var tools = {
     }
 };
 
-function getPointToDraw(position,container,canvasOffset){
+/**
+ * Draws a featurePoint on myCanvas
+ * @param {Event} position - click position
+ * @param {SVGElement} container - shape that should hold the featurePoint
+ * @param {DOMReact | Object} canvasOffset - offset due to canvas
+ * @returns {SVGElement} SVGElement of featurePoint
+ */
+function getPointToDraw(position, container, canvasOffset) {
     var containerOffset = {
-        x: container.parent().attr("x"),
-        y :container.parent().attr("y")
+        x: container.parent().parent().attr("x"),
+        y :container.parent().parent().attr("y")
```

Comment:
Why do we need to call parent() 2 times?

Reply:
We need to call parent two times because the plugin "svg-pan-zoom" nests the `<shape>` tags in a `<g>` tag. The hierarchy looks like this:

```
<svg>
  <g>
    <rect/circle/polygon>
  <g>
</svg>
```

The container actually points to the `<rect/circle/polygon>`. To get the container offset, we need to access the <svg> element.

Reply:
then it may create issue when we don't zoom in/out. I mean the nesting can be change any moment. Can we create a function to get the parent upto desire tag? Eg `parent('svg', container).attr("x")`

rough logic
``` 
if (container.tag !== 'svg' ) container = container.parent();
else return container;
```

Reply:
Done. I understand that my implementation is too heavily reliant on the fact that the plugin nests a `<g>` tag. Hence I have removed that logic and replaced it with a check to get the nearest svg element, which is the parent of the container.